### PR TITLE
[evictor] Initial Evictor release (alpha)

### PR DESCRIPTION
**Pull request by KarolisL**

### CODE REVIEW

**Review by zilvinasu — APPROVED** [review]

File: charts/evictor/templates/role.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
```
Comment (anjmao) [inline charts/evictor/templates/role.yaml:1]: --- can be removed I guess

File: charts/evictor/templates/rolebinding.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (anjmao) [inline charts/evictor/templates/rolebinding.yaml:1]: Here too

File: charts/evictor/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.1.0-alpha2"
```
Comment (anjmao) [inline charts/evictor/Chart.yaml:24]: If we follow semver having alpha suffix on non 1.0.0 version seems unnecessary. Maybe just `0.1.0`

File: charts/evictor/values.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
+# Default values for evictor.
+# This is a YAML-formatted file.
```
Comment (anjmao) [inline charts/evictor/values.yaml:2]: This comment doesn't help :) probably leftover from generated code.

**Review by anjmao — APPROVED** [review]

**Pull request merged**